docs/contributing.rst

```diff
@@ -284,6 +284,34 @@ is required. The steps are roughly:
 - potentially remove the font being replaced from ``package.json`` and re-run ``yarn``
 - commit all of the changed files
 
+.. _translating-the-theme:
+
+Translating the theme
+=====================
+
```

Comment:
Add a short explanation for the structure of our translation files. What is a .po file? What is a .pot file? How do they relate, etc. we can link out to docs for full explanation, but someone should be able to glance here and get an idea quickly

Reply:
ce6ddae5